tests/services/test_path_migration.py

```
@@ -70,38 +68,38 @@ def test_migrate_files_success(self, db_session, tmp_path, file_inventory_factor
-        
+
         # Old directory should be removed if empty
         assert not old_path.exists()
 
     def test_migrate_no_files(self, db_session, tmp_path):
         """Test migration when no files exist."""
         old_path = tmp_path / "old_cold"
         new_path = tmp_path / "new_cold"
-        
+
         success, error, stats = PathMigrationService.migrate_files(
```

Ruff (0.15.7)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)